rollup.config.js

```diff
@@ -5,7 +5,7 @@ import del from 'rollup-plugin-delete'
 import dts from 'rollup-plugin-dts'
 import { typescriptPaths } from 'rollup-plugin-typescript-paths'
 
-import pkg from './package.json' with { type: 'json' }
+import pkg from './package.json' assert { type: 'json' }
```

Comment:
Are you getting an error on something? This syntax is deprecated since node 20 and removed in node 22